platform/features/layout/test/FixedControllerSpec.js

```diff
@@ -157,9 +157,9 @@ define(
                 };
                 testValues = { a: 10, b: 42, c: 31.42 };
                 testConfiguration = { elements: [
-                    { type: "fixed.telemetry", id: 'a', x: 1, y: 1 },
-                    { type: "fixed.telemetry", id: 'b', x: 1, y: 1 },
-                    { type: "fixed.telemetry", id: 'c', x: 1, y: 1 }
+                    { type: "fixed.telemetry", id: 'a', x: 1, y: 1, useGrid: true},
+                    { type: "fixed.telemetry", id: 'b', x: 1, y: 1, useGrid: true},
```

Comment:
Question: when a fixed position has been created before you added this feature, so it's configuration doesn't specify "useGrid".  What happens?

E.g. Checkout master, create fixed position, checkout your branch, view fixed position. Does it work? (I haven't tested).